### ISSUE TITLE
How to extract jpg from mdb files 

### ISSUE THREAD

**Issue by DiiaanaD** [fyu/lsun#8]
Hello,

I just download the bedroom data and right now I have in lsun-folder a bedroom_train_lmdb.zip. I tried to extract jpg from it and using the command "python data.py export bedroom_train_imdb.zip/bedroom_train_lmdb --out_dir bedroom and I receive the following error: lmdb.Error: bedroom_train_lmdb.zip/bedroom_train_lmdb: The system cannot find the path specified.

Same error using:  "python data.py export bedroom_train_imdb.zip --out_dir bedroom 

Could you please give me a hint what I'm doing wrong?

Thanks,
Diana

**Comment by DiiaanaD** [comment]
Later note:

I just extract the archive and now I receive the following error: OverflowError: Integer argument exceeds limit. -file "data.py", line 92, in main

I tried to change to 1099511627776 value and after doing it, I receive "MDB_INVALID: File is not an LMDB file" 

I have no idea how to fix it..

Thanks in advance,
Diana

**Comment by fajarnuha** [comment]
in case you're still wondering, extract your zip file first, then specify the dir in which mdb files are located

**Comment by htwang14** [comment]
I can't even unzip the file using `unzip test_lmbd.zip`

**Comment by YingqingHe** [comment]
> I can't even unzip the file using `unzip test_lmbd.zip`

same problem

**Comment by YingqingHe** [comment]
> I can't even unzip the file using `unzip test_lmbd.zip`

Have you solved this?

**Comment by htwang14** [comment]
> > I can't even unzip the file using `unzip test_lmbd.zip`
> 
> Have you solved this?

Not really. But I found some other repos providing directly downloadable (resized) LSUN test sets (e.g., [here](https://github.com/deeplearning-wisc/vos/tree/main/classification#outlier-datasets)), which serves my purpose. Not sure whether this is helpful for you.

**Comment by YingqingHe** [comment]
> 



> > > I can't even unzip the file using `unzip test_lmbd.zip`
> > 
> > 
> > Have you solved this?
> 
> Not really. But I found some other repos providing directly downloadable (resized) LSUN test sets (e.g., [here](https://github.com/deeplearning-wisc/vos/tree/main/classification#outlier-datasets)), which serves my purpose. Not sure whether this is helpful for you.

Thanks!!!

**Comment by lineeralgebra** [comment]
get rename zip and unzip
